examples/kubernetes/otel-collector-config.yml

```
@@ -22,7 +22,7 @@ receivers:
       # Parse CRI-O format
       - type: regex_parser
         id: parser-crio
-        regex: '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) (?P<log>.*)$'
+        regex: '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*)[ ]{0,1}(?P<log>.*)$'
```

I believe this a shorter equivalent

```suggestion
        regex: '^(?P<time>[^ Z]+) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) ?(?P<log>.*)$'
```

---

I was actually also wondering if maybe `\s*` could be used instead or so

---

I think It would be inconsistent with other places of this regexp where we expect exact one space

---

All of these solutions work, I think " ?" would be best. Thanks for great input everyone.